modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -238,7 +240,7 @@ The `hosts` parameter is a list of separate bare metal assets used to build the
 
 | [[bootMACAddress]]`bootMACAddress`
 |
-| The MAC address of the NIC the host will use to boot on the `provisioning`  network.
+| The MAC address of the NIC the host will use to boot on the `provisioning`  network. The installer will retrieve the IP address using the `bootMACAddress` configuration setting. Ironic binds to the host using the retrieved IP address.
```

Comment:
I noticed that the category names of this chart (Name, Default, Description) are not bolded. Category names in other charts are bolded.

Reply:
I added the header tag. It seems to be bolded in my preview. Let me know if the header tag doesn't work.